### PR TITLE
Add a day2 repetition slide

### LINKED ISSUE #118: Create template for day 2 repetition/exercises
I think it's pretty cool to start into day 2 with a few exercises to jog people's memory and wake 'em up.
So, what we had so far was a modified version of this base template:

```
1. use `cd` to leave the training repository
1.1 make new local repository

2. add a new local name and email in gitconfig
2.1 verify local config

3. add a readme and commit
3.1 note commit id
3.2 make changes in readme and amend commit
3.3 compare commit ids

4. create a feature/testdata branch
4.1. switch to branch

5. make directory called 'testdata'
5.1 check status
5.2 add empty file called .keep in testdata
5.3 check status again
5.4 commit new directory

6. fill testdata with a file and commit

7. go to main branch and commit edits in readme

8. cherry-pick (with the -x flag) the first commit from feature/testdata
8.1 check git log

9. add .gitignore that ignores the testdata and commit
9.1. check tig

10. git merge (with the --no-ff flag) the feature/testdata branch
10.1. check tig again
```

We should totally go over it and collab on a new version though.

### PR DESCRIPTION
**Pull request by martialblog**
Added before the workflow slides.

Tried a bullet point list, but it didn't fit on the page. Suggestions are welcome

Fixes #118

### CODE REVIEW

**Comment by theFeu** [comment]
Currently at the wrong laptop and can't test it atm, @htriem can you hava a look?

**Review by htriem — CHANGES_REQUESTED** [review]
This looks good in principle - looking at the presenter mode, I'm not too sure about the font size though.
I'd be worried about showing this on a badly lit small screen. 

Is a side by side arragement possible, or can we have a higher contrast between the text and text background?

**Comment by theFeu** [comment]
We can't really change anything about the style, since it's global for all trainings...

What we could do is split it on two slides, so we have 2 smaller rounds of exercises?

**Comment by theFeu** [comment]
This should be a bit more readable, what do you think @martialblog @htriem ?

**Comment by martialblog** [comment]
If we split it into two slides, then bullet points might fit on the pages. I just used the Code Block since the bullet points were spaced too wide

**Comment by theFeu** [comment]
image

Fits well enough I suppose?

It doesn't look like you can group them together - which bothers me a little - but I can't think of anything better atm...

File: day2/00_Repetition/01_Intro.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+   * Add an empty file called `.keep` in `testdata/`
+   * Check the git status again
+   * Commit the new directory
+    
+    
+!SLIDE smbullets small noprint
+
+## Advanced Commands
+
+   * Switch to the master branch
```
Comment (martialblog) [inline day2/00_Repetition/01_Intro.md:28]: TODO: change to `main`

**Review by martialblog — COMMENTED** [review]
I think the bullet points look better.

**Comment by martialblog** [comment]
Maybe we can try a numbered list?

**Comment by theFeu** [comment]
image
 
I think this looks good

**Comment by theFeu** [comment]
Okay, I am happy with this.
I think this can be squashed and merged

**Review by martialblog — COMMENTED** [review]
:+1: LGTM squash and merge

**Pull request merged**